inspirehep/dojson/hep/fields/bd90x99x.py

```diff
@@ -32,15 +32,36 @@
 @utils.filter_values
 def references(self, key, value):
     """Produce list of references."""
+    recid = ''
+    number = ''
+    year = ''
+    if '0' in value:
+        try:
+            recid = int(value.get('0'))
+        except:
+            # Some crap in they recid :-(
```

Comment:
Right, can you amend it to use the emoji :crap:. It's definitively sweeter :smile:.

(my fault sorry. but the input is really bad sometimes...)